File: ABC_py/02_ABC081A_Placing_Marbles.py
# 問題: https://atcoder.jp/contests/abs/tasks/abc081_a
# 解説 https://delta114514.hatenablog.jp/entry/2018/03/15/014555

print(input().count('1'))

# 素直にCから変換するとこうなる
'''
s = list(input())
count = 0
for i in range(3):
    if s[i] == '1':
        count += 1
print(count)
'''

File: ABC_cpp/02_ABC081A_Placing_Marbles.cpp
// 問題: https://atcoder.jp/contests/abs/tasks/abc081_a
// 解説: https://qiita.com/drken/items/fd4e5e3630d0f5859067

#include <bits/stdc++.h>
using namespace std;

int main()
{
    string masu;
    cin >> masu;

    int count = 0;
    if (masu[0] == '1') count++;
    if (masu[1] == '1') count++;
    if (masu[2] == '1') count++;
    cout << count << endl;
    return 0;
}
